subprojects/performance/src/performanceTest/groovy/org/gradle/performance/experiment/java/ParallelBuildPerformanceTest.groovy

```diff
@@ -44,8 +44,8 @@ class ParallelBuildPerformanceTest extends AbstractCrossBuildPerformanceTest {
 
         where:
         testProject                  | memory | warmUpRuns | runs
-        "largeMonolithicProjectJava" | '608m' | 2          | 6
-        "largeMultiProjectJava"      | '608m' | 2          | 6
+        "largeMonolithicJavaProject" | '4g'   | 2          | 6
```

Comment:
For such large projects we should use forked compilation instead and keep the main daemon small.